scripts/ci/gitlab/pipeline/publish.yml

```
@@ -147,3 +147,12 @@ publish-draft-release:
   script:
     - ./scripts/ci/gitlab/publish_draft_release.sh
   allow_failure:                   true
+
+update-node-template:
+  stage:                           publish
```

```suggestion
  stage:                           publish
  extends:                         .kubernetes-env
```
It's also needed to specify a runner. Here I suggest k8s runners because it doesn't require heavy rust compilation.